src/firmware/app/world/charger.h

```diff
@@ -24,27 +24,27 @@ Charger_t* app_charger_create(void (*charge_capacitor)(void),
  * NOTE: This will not destroy the values pointed to by any pointers passed to the
  *       `create` function
  *
- * @param charger The charger to destroy
+ * @param charger [in] The charger to destroy
```

Comment:
I think this is also an exception since Charger_t* is a class

Reply:
Done.